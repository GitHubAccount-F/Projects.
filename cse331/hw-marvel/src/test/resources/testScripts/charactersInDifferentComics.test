#Tests when the path involves searching through multiple comics,
#but is a linear search(characters connecting to other characters
#until we directly end up at the target)

#Creates graph
LoadGraph graph1 directPathNotInSameComics.csv

#Finds path between characters
FindPath graph1 SpiderMan CaptainAmerica

FindPath graph1 Venom Thor

FindPath graph1 IronMan IronMan