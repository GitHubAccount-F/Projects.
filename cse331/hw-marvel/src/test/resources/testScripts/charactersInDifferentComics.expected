#Tests when the path involves searching through multiple comics,
#but is a linear search(characters connecting to other characters
#until we directly end up at the target)

#Creates graph
loaded graph graph1

#Finds path between characters
path from SpiderMan to CaptainAmerica:
SpiderMan to Venom via Spider-Verse
Venom to IronMan via Stark-Tower-Falling
IronMan to Thor via Ragnarok
Thor to CaptainAmerica via Civil-War

path from Venom to Thor:
Venom to IronMan via Stark-Tower-Falling
IronMan to Thor via Ragnarok

path from IronMan to IronMan: